app/www/profile/customize/index.php

```diff
@@ -0,0 +1,41 @@
+<?php require $_SERVER['DOCUMENT_ROOT']."/includes/scaffolder.php";
+head('Your Avatar', 4);
+$user = $_SESSION['user'];
+?>
+<div class="portal-body flexbox">
+  <div class="profile-avatar">
+    <img class="profile-avatar-img" src="/images/avatar/<?php echo $user['avatar'];?>.svg"/>
+  </div>
+  <span class="portal-body flexbox">
```

Comment:
There should only be one element with the class `portal-body`; I think if you remove that here and leave it as just a `flexbox`, you'll get the view you want